src/neuroglancer/viewer.ts

```diff
@@ -696,11 +696,61 @@ export class Viewer extends RefCounted implements ViewerState {
     this.registerDisposer(registerActionListener(this.element, action, handler));
   }
 
+  /**
+   * Called after an edit is made to "deactivate" merge/split
+   */
+   deactivateEditMode() {
+    if (this.mouseState.actionState === ActionState.INACTIVE) {
+      this.mouseState.toggleAction();
+    }
+    this.handleModeChange(this.mouseState.actionMode === ActionMode.MERGE);
+  }
+
+  handleModeChange(merge: boolean) {
```

Comment:
I left that code in there because I wasn't sure the direction to go for to handle graphene specific UI. I haven't touched it since adapting to your recent changes.

Also I I forgot to mention merge and split. We send the xyz coordinates to the backend for the merge and split points. We also have our multicut feature were we can select multiple points for a and b so our server can compute a better split point. So you would suggest we implement this as additional tools rather than change/add to the existing split+merge methods?

Reply:
didn't see your above comment, reading that now...

Reply:
It seems like merge could potentially be modified if the only change is the addition of the position --- it seems like it might be reasonable to always provide the position, and graph sources that don't care can just ignore it.  For split it would probably make more sense to create a separate tool since there are a lot of differences.  Then the SegmentationGraphSource could indicate which types of splits it supports.

To provide the position, you will need a RenderLayerTransform associated with the SegmentationGraphSourceConnection  so that the position can be transformed to the appropriate coordinate space --- you could add a `transform: WatchableValueInterface<RenderLayerTransformOrError>` parameter to `SegmentationGraphSource.connect`.